internal/provider/resource_gitlab_topic_test.go

```diff
@@ -256,6 +258,46 @@ func TestAccGitlabTopic_softDestroy(t *testing.T) {
 	})
 }
 
+func TestAccGitlabTopic_titleSupport(t *testing.T) {
+	rInt := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabTopicDestroy,
+		Steps: []resource.TestStep{
+			{
+				SkipFunc: isGitLabVersionAtLeast(context.Background(), testGitlabClient, "15.0"),
```

Comment:
@timofurrer  - since every step is using the SkipFunc, I'd suggest we move this to a pre-check.